hyperopt/criteria.py

```diff
@@ -70,9 +72,9 @@ def logEI_gaussian(mean, var, thresh):
             nonnegs = np.logical_not(negs)
             negs_score = score[negs]
             negs_pdf = n.logpdf(negs_score)
-            r = np.exp(np.log(-negs_score)
-                       + n.logcdf(negs_score)
-                       - negs_pdf)
+            r = np.exp(np.log(-negs_score) +
+                       n.logcdf(negs_score) -
+                       negs_pdf)
```

Comment:
yes, it's a linter thing (e.g. both flake8 and pep8 require it). I'm personally agnostic w.r.t. readability, but it's safer for weird interpreter setups, since a binary operator before the break signals continuation. I've stuff like this fail a few times. Another solution would be to wrap it in `()` or append a `\` after each break, over both of which I'd favour the current one.